docs/examples/nested_regions.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region, Page
+from selenium.webdriver.common.by import By
+
+
+class MainPage(Page):
+
+    @property
+    def menu1(self):
+        root = self.find_element(By.ID, "menu1")
+        return Menu(self, root=root)
+
+    @property
+    def menu2(self):
+        root = self.find_element(By.ID, "menu2")
+        return Menu(self, root=root)
+
+
+class Menu(Region):
+
+    @property
+    def entries(self):
+        items = self.find_elements(*Entry._root_locator)
```

Comment:
You shouldn't use `_root_locator` with repeated regions. I would have an `_entries_locator` as a class variable. By defining a `_root_locator`, the behaviour will be different and you may get unexpected elements.

Reply:
OK, I understand why this would lead to problems. Does that imply that a `_root_locator` should be able to uniquely identify an element inside a page or region?

Reply:
OK, just read your last comment about uniqueness.